### ISSUE TITLE
Patched?

### ISSUE THREAD

**Issue by Tibzzy** [doug157/AimGods#2]
Is it patched?

Hitting a player more than once in 1 bullet causes a kick, along with name spoofing not working anymore. Not only that but the game kicks you out mid match even if you haven't shot a bullet (checking if the anticheat is running?).

**Comment by doug147** [comment]
Yup they patched the 1 shot kill last night and the name spoofing

**Comment by Tibzzy** [comment]
> Yup they patched the 1 shot kill last night and the name spoofing

You get kicked out now mid match even without using name spoofing and the 1 bullet kill. Does the same happen to you?

**Comment by SubstituteR** [comment]
> > Yup they patched the 1 shot kill last night and the name spoofing
> 
> You get kicked out now mid match even without using name spoofing and the 1 bullet kill. Does the same happen to you?

That will always happen with EAC. If you don't want to get kicked, run EAC with a proper bypass.
